includes/class.llms.template.loader.php

```diff
@@ -28,9 +28,18 @@ class LLMS_Template_Loader {
 	 * @since 1.0.0
 	 * @since 3.20.0 Unknown.
 	 * @since 3.41.1 Predispose posts content restriction in REST requests.
+	 * @since [version] Handle block templates loading.
 	 */
 	public function __construct() {
 
+		// Template loading for FSE themes.
+		add_action(
```

Comment:
I don't want us to start using anonymous functions for hooks.I don't have any really good reason for this but it *feels* like it's a really bad pattern to start implementing, unless there's a really good reason for it. This seems like the reason is "I don't want to add a callback that only adds a filter" and that doesn't feel like a good reason for it in my opinion.

Reply:
Well yeah, I thought that isn't a bad pattern and couldn't think why not using it in this specific case. :D 
Why is it a bad pattern? I think it is bad when you might want to unhook a callback, but:
1) in this specific case you wouldn't be able to unhook the `template_redirect` callback from outside this class in any case as the object is instantiated on file loading
2) if it wasn't for 1) you could always unhook the `pre_get_block_templates` callback hooking your "unhooking logic" at `template_redirect|11`.
That was my reasoning: no reason to not do that, and gold.

That said, I'm going to follow your feeling <3